miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java

```diff
@@ -99,6 +100,10 @@
   private ContainerService containerService;
   @Autowired
   private BoxService boxService;
+  @Value("${miso.error.edit.distance:2}")
```

Comment:
I kind of hate that the defaults get smeared everywhere, but no suggestions for a better way, so, :man_shrugging:

Reply:
I'm going to extract it into a microcontroller because we need to use the logic outside of just the display.